frontend/app/helpers/API.js

```diff
@@ -88,18 +88,22 @@ export default {
     return post(`/api/send/campaignUpdate/${id}/`)
   },
 
-  newCampaignUpdate: function(id, data, cb = response => response) {
-    return post(`/api/campaigns/${id}/update/new`, data).then(data => cb(data))
+  newCampaignAction: function(id, data, cb = response => response) {
+    return post(`/api/campaigns/${id}/action/new`, data).then(data => cb(data))
   },
 
-  campaignCall: function(id, cb = response => response) {
```

Comment:
nvm, I see, we're consolidating the `CampaignCall` and `CampaignUpdate` endpoints into one `CampaignAction` endpoint